src/raftstore/store/fsm/peer.rs

```diff
@@ -608,7 +632,7 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
             .map(move |_| {
                 fail_point!(
                     "on_raft_log_gc_tick_1",
-                    peer_id == 1 && tick == PeerTick::RaftLogGc,
+                    peer_id == 1 && tick == PeerTicks::RAFT_LOG_GC,
                     |_| unreachable!()
                 );
                 if let Err(e) = mb.force_send(PeerMsg::Tick(tick)) {
```

Comment:
maybe add a comment to indicate that it's fine to not clean registry here cause the error happens when the node is shutting down